envs/postgres-s3-external-spark/docker-compose.yml

```diff
@@ -22,20 +22,20 @@ services:
 
 
   s3server:
-    build:
-      context: ${DOCKER_CONTEXT_FOLDER}/patched-localstack
```

Comment:
Is it still patched?

Reply:
Nope. This is the removed lines part ;-)